enhancements/kubelet/user-namespaces-support.md

```diff
@@ -0,0 +1,435 @@
+```
+       // userNamespaceLevel determines if the policy allows host users in containers.
+       // Valid values are "AllowHostLevel", "RequirePodLevel", and omitted.
+       // When "AllowHostLevel" is set, a pod author may set `hostUsers` to either `true` or `false`.
+       // When "RequirePodLevel" is set, a pod author must `hostUsers` to `false`.
+       // When omitted, this means no opinion and the platform is left to choose a reasonable default,
+       // which is subject to change over time.
+       // The current default is "AllowHostLevel".
+       // +openshift:enable:FeatureGate=UserNamespacesPodSecurityStandards
+       // +kubebuilder:validation:Enum="AllowHostLevel";"RequirePodLevel";""
+       // +optional
+       UserNamespaceLevel NamespaceLevelType `json:"userNamespaceLevel" protobuf:"bytes,26,opt,name=userNamespaceLevel"`
+...
+// NamespaceLevelType shows the allowable values for the UserNamespaceLevel field.
+type NamespaceLevelType string
+...
+const (
+       // NamespaceLevelAllowHost allows a pod to set `hostUsers` field to either `true` or `false`
+       NamespaceLevelAllowHost NamespaceLevelType = "AllowHostLevel"
+       // NamespaceLevelRequirePod requires the `hostUsers` field be `false` in a pod.
+       NamespaceLevelRequirePod NamespaceLevelType = "RequirePodLevel"
+       // NamespaceLevelEmpty will set to the default, which is currently "AllowHostLevel".
+       NamespaceLevelEmpty NamespaceLevelType = ""
+
+...
+)
+
+```
+
+This field branches from its fellow `AllowHost*` namespace fields to follow current API conventions, which maintain boolean fields are not extendable enough.
+
+##### restricted-v3
+
+This SCC profile will be identical to the existing `restricted-v2`, except it will set the `UserNamespaceLevel` to `RequirePodLevel`,
+thus forcing pods to be in a user namespace. This will make it a more restrictive profile, as the user on the host will not be
+the same as the one inside the container.
+
+Note: upstream Kubernetes deemed `runAsUser` to be a safe field to use for pods with `hostUsers: false`. However, the philosophy
+of SCC utilizes defense-in-depth. `restricted-v3` will be the most restrictive policy.
+
+##### nonroot-v3
+
+This SCC profile will be similar to `nonroot-v2`, except it will set `UserNamespaceLevel` to `RequirePodLevel`, thus forcing pods to be in a
+user namespace, and set the fields `runAsUser`/`runAsGroup`/`fsGroup` to `runAsAny` (in this way, it's similar to `anyuid` SCC, but without `allowPrivilegeEscalation` set to `true`).
+
+Note: a user can actually request `runAsUser: 0` in this profile, despite its name being `nonroot`. This is because root inside
+the container is not root on the host, and because `anyuid` SCC allows privilege escalation, which we deemed out of scope for this SCC.
+
+From a pod ability perspective, the goal of `nonroot-v3` is to give a better security profile as `restricted-v2`, but with more capabilities. On the host, the container user will be
```

Comment:
which additional capabilites?  The description doesn't mention a change and I thought nonroot-v2 and restricted-v2 had the same `allowedCapabilities`, `defaultAddCapabilities`, and `requiredDropCapabilities`